### PR TITLE
Issue #37. Add "menu main navigation" and "hamburger main navigation for mobile display". 

### PR DESCRIPTION
**Pull request by alvincases**

### CODE REVIEW

**Comment by alvincases** [comment]
Menu links could not exported @marychris created new ticket for this https://github.com/Luukyb/drupalcampcebu2018/issues/46.

File: drupal/web/sites/default/services.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    #   about template variables.
+    # - Twig templates are automatically recompiled whenever the source code
+    #   changes (see auto_reload below).
+    #
+    # For more information about debugging Twig templates, see
+    # https://www.drupal.org/node/1906392.
+    #
+    # Not recommended in production environments
+    # @default false
+    debug: true
```
Comment (marychris) [inline drupal/web/sites/default/services.yml]: Set this back to false. (debug: true) This should be false by default for production.
Find development.services.yml, there set (debug: true) to false.
At the end of settings.local.php add 
'$settings['container_yamls'][] = DRUPAL_ROOT . '/sites/development.services.yml';'

File: drupal/web/themes/custom/drupalcamp_cebu_2018/templates/navigation/menu--main.html.twig [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+  {% if items %}
+    {% if menu_level == 0 %}
+   <div class="ds-menu-wrapper" >
+      <ul{{ attributes.addClass('list-inline') }}>
+    {% else %}
+      
+      <ul class="list-inline">
```
Comment (marychris) [inline drupal/web/themes/custom/drupalcamp_cebu_2018/templates/navigation/menu--main.html.twig]: Double check if we can remove <ul class="list-inline"> or not? 
The class is already being added in '<ul{{ attributes.addClass('list-inline') }}>'

File: drupal/web/themes/custom/drupalcamp_cebu_2018/templates/navigation/menu--hamburger-main-navigation.html.twig [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+  {% if items %}
+    {% if menu_level == 0 %}
+    <span class="hamburger" onclick="openNav()">&#9776;</span>
+    <div id="side-navigation" class="menu-wrapper">
+      <div class="close-button col-2 col-sm-3 col-md-2"><a href="javascript:void(0)" onclick="closeNav()">&times;</a></div>
+      <ul{{ attributes.addClass('list-group') }}>
+    {% else %}
+      <ul class="list-group">
```
Comment (marychris) [inline drupal/web/themes/custom/drupalcamp_cebu_2018/templates/navigation/menu--hamburger-main-navigation.html.twig]: Check if we can remove <ul class="list-group"> or not. 
Class list-group is already being added in <ul{{ attributes.addClass('list-group') }}>.

**Pull request merged**